.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1738,26 +1373,70 @@ public class FineTuningClient {
         public virtual ClientResult GetJobs(string after, int? limit, RequestOptions options);
         public virtual Task<ClientResult> GetJobsAsync(string after, int? limit, RequestOptions options);
     }
+    public class FineTuningJob {
+        public DateTimeOffset CreatedAt { get; }
+        public FineTuningJobError Error { get; }
+        public DateTimeOffset? EstimatedFinish { get; }
+        public string FineTunedModel { get; }
+        public DateTimeOffset? FinishedAt { get; }
+        public FineTuningJobHyperparameters? Hyperparameters { get; set; }
+        public string Id { get; }
+        public string Model { get; }
+        public string Object { get; }
+        public string OrganizationId { get; }
+        public IReadOnlyList<string> ResultFiles { get; }
+        public int Seed { get; }
+        public FineTuningJobStatus Status { get; }
+        public int? TrainedTokens { get; }
+        public string TrainingFile { get; }
+        public string ValidationFile { get; }
+    }
+    public class FineTuningJobError {
+        public string Code { get; }
+        public string Message { get; }
+        public string Param { get; }
+    }
+    public partial struct FineTuningJobHyperparameters {
+        private object _dummy;
+        private int _dummyPrimitive;
+        public FineTuningJobHyperparameters(BinaryData nEpochs, IDictionary<string, BinaryData> serializedAdditionalRawData = null);
+        public FineTuningJobHyperparameters(BinaryData nEpochs);
+        public BinaryData BatchSize { get; }
+        public BinaryData LearningRateMultipler { get; }
+        public BinaryData NEpochs { get; }
```

Comment:
Each of these is a weird union type that needs massaging and a rename -- I think it sets the record for highest concentration! We can use something like `ChatToolChoice` as a pattern to match (and I can follow up about customizing the code generation input the same way we did for that type -- this doesn't block wiring it up but will improve how much of it can be generated).

We probably want all of these to be possible (making some names up on the fly):
```csharp
options.BatchSizeSelection = HyperparameterBatchSizeSelection.Auto;
options.BatchSizeSelection = new HyperparameterBatchSizeSelection(42);
options.BatchSizeSelection = 42; // via implicit operator
options.LearningRateMultiplierSelection = HyperparameterLearningRateMultiplierSelection.Auto;
options.LearningRateMultiplierSelection = new(1.5); // no clue what real values are!
options.LearningRateMultiplierSelection = 1.5; // again via implicit operator
// This could be something like "EpochCycleCountSelection" or another name with "Epoch" in it if we can make it fit; we just really need the N to turn into a Count somewhere
options.CycleCountSelection = HyperparameterCycleCountSelection.Auto;
options.CycleCountSelection = new(5);
options.CycleCountSelection = 5;
```